src/Sarif.Converters/PylintObjectModel/PylintLog.cs

```diff
@@ -0,0 +1,9 @@
+﻿
```

Comment:
There are still several source files in this PR without copyright notices.

Reply:
Missed it. Will fix it.